packages/typespec-client-generator-core/src/types.ts

```diff
@@ -1089,16 +1104,68 @@ export function getSdkModelPropertyType(
       })
     );
   }
-  return diagnostics.wrap({
+  const multiPartOption: MultipartOptionsType = {
+    isNameDefined: httpOperationPart ? httpOperationPart.name !== undefined : true,
+    isFilePart: isBytesInput && operationIsMultipart,
+    multi: httpOperationPart ? httpOperationPart.multi : isRawBytesArray,
+    headers: [], // TODO
```

Comment:
why expose the API when it is not implemented?

Reply:
removed